constants.js

```diff
@@ -0,0 +1,8 @@
+export const DEFAULT_TSCONFIG_LOCATIONS = [
+  'tsconfig.eslint.json',
+  'tsconfig.json'
+]
+
+export const DEFAULT_EXTENSIONS = ['js', 'jsx', 'mjs', 'cjs', 'ts', 'tsx']
+
+export const CURRENT_WORKING_DIRECTORY = process.cwd()
```

Comment:
Currently, we dropped support for `--cwd` flag and option as other `standard` engines doesn't implement it, if we want to have this option, we should implement it in `standard-engine`, for the moment, we use `process.cwd()`.